README.md

```diff
@@ -1479,25 +1479,6 @@ To run the script, activate your virtual environment, then run
 python scripts/apidev.py
 ```
 The script gives you a url where your current version of the documentation
-is visible. Change the docs in `azul/service/app.py`, save, and refresh the page and
-your changes will appear immediately along with any warnings or errors they introduce.
-
-Some changes to `app.py` won't take effect with this script unless the changes
-are deployed in the active deployment. Specifically, changes to
-
-- the global OpenAPI spec
-- the OpenAPI spec within the `spec` parameter to the `route` decorator
-- the *body* of pre-existing route functions
-
-will **not** require redeployment. Any other changes **will**, such as
-
-- adding/removing routes
```

Comment:
This still appears to be the case. When I tried deleting or adding routes in `app.py`, these changes were not reflected in the swagger editor. It printed "spec updated" twice and did not show my changes even after refreshing the page and re-starting the script.

Reply:
I experienced the problem you mentioned where updates where not registered. While trying to debug it, I somehow fixed it, because now I am unable to reproduce it.

Regardless, the comment _is_ still accurate.